packages/rocketchat-reactions/setReaction.js

```diff
@@ -41,8 +44,10 @@ Meteor.methods({
 			if (_.isEmpty(message.reactions)) {
 				delete message.reactions;
 				RocketChat.models.Messages.unsetReactions(messageId);
```

Comment:
Can you add the callbacks to run on the client side as well? This way people can take advantage of it for statistics, such as Piwiki or future Google Analytics